### ISSUE TITLE
error during Xcode build with new iOS version

### ISSUE THREAD

**Issue by awaik** [derTuca/flutter_google_places_picker#33]
### Error description

Works fine with Android and worked before with iOS. Now, after updates of the Flutter and XCode gives errors:

```
Failed to build iOS app
Error output from Xcode build:
↳
    ** BUILD FAILED **


Xcode's output:
↳
    Command CompileSwift failed with a nonzero exit code
    /Users/macbook/development/flutter/.pub-cache/hosted/pub.dartlang.org/google_places_picker-2.1.0+2/ios/Classes/GooglePlacesPickerPlugin.m:82:36: error: property 'autocompleteBoundsMode' not found on object of type 'GMSAutocompleteViewController *'
                autocompleteController.autocompleteBoundsMode = kGMSAutocompleteBoundsModeRestrict;
                                       ^
    /Users/macbook/development/flutter/.pub-cache/hosted/pub.dartlang.org/google_places_picker-2.1.0+2/ios/Classes/GooglePlacesPickerPlugin.m:82:61: error: use of undeclared identifier 'kGMSAutocompleteBoundsModeRestrict'
                autocompleteController.autocompleteBoundsMode = kGMSAutocompleteBoundsModeRestrict;
                                                                ^
    /Users/macbook/development/flutter/.pub-cache/hosted/pub.dartlang.org/google_places_picker-2.1.0+2/ios/Classes/GooglePlacesPickerPlugin.m:93:32: error: property 'autocompleteBounds' not found on object of type 'GMSAutocompleteViewController *'; did you mean 'autocompleteFilter'?
            autocompleteController.autocompleteBounds = bounds;
                                   ^~~~~~~~~~~~~~~~~~
                                   autocompleteFilter
    In module 'GooglePlaces' imported from /Users/macbook/development/flutter/.pub-cache/hosted/pub.dartlang.org/google_places_picker-2.1.0+2/ios/Classes/GooglePlacesPickerPlugin.h:2:
    /Users/macbook/projects/GoodLuckApp/ios/Pods/GooglePlaces/Frameworks/GooglePlaces.framework/Headers/GMSAutocompleteViewController.h:120:63: note: 'autocompleteFilter' declared here
    @property(nonatomic, strong, nullable) GMSAutocompleteFilter *autocompleteFilter;
                                                                  ^
    /Users/macbook/development/flutter/.pub-cache/hosted/pub.dartlang.org/google_places_picker-2.1.0+2/ios/Classes/GooglePlacesPickerPlugin.m:93:51: warning: incompatible pointer types assigning to 'GMSAutocompleteFilter * _Nullable' from 'GMSCoordinateBounds *' [-Wincompatible-pointer-types]
            autocompleteController.autocompleteBounds = bounds;
                                                      ^ ~~~~~~
    1 warning and 3 errors generated.
    Command MergeSwiftModule failed with a nonzero exit code
    note: Using new build system
    note: Building targets in parallel
    note: Planning build
    note: Constructing build description
```

### Flutter doctor

```
[✓] Flutter (Channel stable, 1.22.0, on macOS 11.0 20A5384c, locale en-RU)
    • Flutter version 1.22.0 at /Users/macbook/development/flutter
    • Framework revision d408d302e2 (3 weeks ago), 2020-09-29 11:49:17 -0700
    • Engine revision 5babba6c4d
    • Dart version 2.10.0

 
[✓] Android toolchain - develop for Android devices (Android SDK version 29.0.3)
    • Android SDK at /Users/macbook/Library/Android/sdk
    • Platform android-30, build-tools 29.0.3
    • Java binary at: /Applications/Android Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b3-6222593)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS (Xcode 12.0.1)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 12.0.1, Build version 12A7300
    • CocoaPods version 1.9.3

[✓] Android Studio (version 4.0)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin version 50.0.1
    • Dart plugin version 193.7547
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b3-6222593)
```

### The plugin version

`google_places_picker: ^2.1.0+2`

**Comment by awaik** [comment]
the temporary workaround - comment 3 lines in the file .../flutter/.pub-cache/hosted/pub.dartlang.org/google_places_picker-2.1.0+2/ios/Classes/GooglePlacesPickerPlugin.m

```
  if (![boundsDictionary isEqual:[NSNull null]] || ![restriction isEqual:[NSNull null]]) {
        double neLat;
        double neLng;
        double swLat;
        double swLng;
        
        if (![restriction isEqual:[NSNull null]]) {
            neLat = [restriction[@"northEastLat"] doubleValue];
            neLng = [restriction[@"northEastLng"] doubleValue];
            swLat = [restriction[@"southWestLat"] doubleValue];
            swLng = [restriction[@"southWestLng"] doubleValue];
        //    autocompleteController.autocompleteBoundsMode = kGMSAutocompleteBoundsModeRestrict;
        } else {
            neLat = [boundsDictionary[@"northEastLat"] doubleValue];
            neLng = [boundsDictionary[@"northEastLng"] doubleValue];
            swLat = [boundsDictionary[@"southWestLat"] doubleValue];
            swLng = [boundsDictionary[@"southWestLng"] doubleValue];
        }
        CLLocationCoordinate2D neCoordinate = CLLocationCoordinate2DMake(neLat, neLng);
        CLLocationCoordinate2D swCoordinate = CLLocationCoordinate2DMake(swLat, swLng);
        
        //GMSCoordinateBounds *bounds = [[GMSCoordinateBounds alloc] initWithCoordinate:neCoordinate coordinate:swCoordinate];
        //autocompleteController.autocompleteBounds = bounds;
        
    }
```

**Comment by niypoo** [comment]
I'm facing this issue too !!!

**Comment by derTuca** [comment]
Sorry for the delay, fixed with 2.1.0+3.
